### ISSUE TITLE
Live stream not playing from seeked postion

### ISSUE THREAD

**Issue by vksbansal** [videojs/video.js#5921]
## Description
Briefly describe the issue.

Stream Info: Live stream with 2 hours of DVR recording **.m3u8**

Issue: Not playing from the location where it is seeking (when paused) (only first time)

## Steps to reproduce
1. pause live `video player.pause()`
2. seek to some point `player.currentTime(seekPoint);`
3. play the video `player.play()`
4. video will start to play from the live location, not from the seek point.

This happens only the first time, subsequent works fine.

Please help, please.... its a major bug.

**Packages:**
"video.js": "^7.4.1",
"@videojs/http-streaming": "^1.9.0",

**Comment by gkatsev** [comment]
It's possible that you're trying to seek too early before we can accurately respond to it. Are you seeking immediately after the source is set?

**Comment by vksbansal** [comment]
@gkatsev actually this will happen even if you do the mentioned steps after some minutes of playback

**Comment by gkatsev** [comment]
My guess would be that you're seeking outside of the seekable range, in that case.
Have a live test case?

**Comment by vksbansal** [comment]
@gkatsev if you can provide me with your email, then I can mail you the required stream to test. and this happens only when 
```

pause live video player.pause()
seek to some point player.currentTime(seekPoint);
play the video player.play()
video will start to play from the live location, not from the seek point.


```which means its only our of range error

**Comment by vksbansal** [comment]
@gkatsev, please help

**Comment by alsotang** [comment]
I meet this problem too. My live stream is a m3u8 file with latest 3 ts file. And when it's playing, videojs continually fetch the m3u8 file for a few seconds. 
When played say 60 seconds, and I seek to 0 potision. The stream would still play from the first ts of the m3u8 file.
I try to debug what happened and found that videojs has a fix seekable logic. Whenever I seek to previous time, it tell me the result of seekable only accept recent time.
But I install a Chrome extension https://chrome.google.com/webstore/detail/play-hls-m3u8/ckblfoghkjhaclegefojbgllenffajdc this one. When playing the same m3u8 file, it can seek correctly. Seems the extension is based on hls.js.

My m3u8 file is http://timeshift.bizlive.myqcloud.com/timeshift/live/f9a7601d79e7fdb9d486c1417272cce6/timeshift.m3u8?delay=100

**Comment by vksbansal** [comment]
Now, when you have provided a test stream @alsotang, I hope someone from the videojs team will revert... its long since I have reported this issue

**Comment by vksbansal** [comment]
this issue has been fixed in 7.6.0
